src/Microsoft.Health.Dicom.Core/Modules/ServiceModule.cs

```diff
@@ -144,6 +144,11 @@ public void Load(IServiceCollection services)
                     .AsSelf()
                     .AsImplementedInterfaces();
 
+                services.Add<GetCustomTagsService>()
```

Comment:
Was it that this service was missing in the DI?

Reply:
yes